catboost/docs/en/_includes/work_src/reusage/r__group-and-group-weight__resctriction.md

```diff
@@ -1,7 +1,7 @@
 
 {% note alert %}
 
-{% include [group-params-python__group-and-group-weight__resctriction__intro](python__group-and-group-weight__resctriction__intro.md) %}
+{% include [group-params-python__group-and-group-weight__restriction__intro](python__group-and-group-weight__resctriction__intro.md) %}
```

Comment:
please rename `catboost/docs/en/_includes/work_src/reusage/python__group-and-group-weight__resctriction__intro.md`

Reply:
sorry, i don't see any related commits